var/spack/repos/builtin/packages/libpressio-tthresh/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class LibpressioTthresh(CMakePackage):
+    """A tthresh implementation for libpressio"""
+
+    homepage = "https://github.com/robertu94/libpressio_tthresh"
+    url = "https://github.com/robertu94/libpressio_tthresh/archive/refs/tags/0.0.1.tar.gz"
+    git = homepage
```

Comment:
(broken git url)

Reply:
Works for me?  Was this meant for another repo?

```
[runderwood@defiant tmp]$ git clone https://github.com/robertu94/libpressio_tthresh
Cloning into 'libpressio_tthresh'...
remote: Enumerating objects: 63, done.
remote: Counting objects: 100% (63/63), done.
remote: Compressing objects: 100% (42/42), done.
remote: Total 63 (delta 22), reused 59 (delta 18), pack-reused 0
Receiving objects: 100% (63/63), 1.45 MiB | 19.75 MiB/s, done.
Resolving deltas: 100% (22/22), done.
[runderwood@defiant tmp]$ curl -LO https://github.com/robertu94/libpressio_tthresh/archive/refs/tags/0.0.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1490k    0 1490k    0     0  3398k      0 --:--:-- --:--:-- --:--:-- 3398k
```